### DISCUSSION TITLE
Menu Entries do not work, but same command from shell works.

### DISCUSSION THREAD

**Discussion by rweisman** [ipxe/ipxe#864] [Q&A] [answered]
I am trying to convert to using iPXE menus.  I am using snponly.

**Comment by mcb30** [comment, marked as answer]
@rweisman You have not provided any information that would allow anyone to help you.  You need to describe the problem that you are having: what are you trying to do, what (precisely) have you tried doing, what is not working?

Marking this as answered and locked since there is no question to answer.  Please open a new discussion with the relevant information if you would like someone to help you.
